Documented init script is no longer valid

It appears that the documentation needs to be updated with the latest release.  The sample init script no longer builds using Typescript.  We are updating from 1.5.0 so not sure when this was introduced.

ERROR in app/app.component.ts:42:9 - error TS2322: Type '"us"' is not assignable to type 'Datacenter'.
42         datacenter: 'us',
           ~~~~~~~~~~

  ../node_modules/@datadog/browser-core/cjs/configuration.d.ts:38:5
    38     datacenter?: Datacenter;
           ~~~~~~~~~~
    The expected type comes from property 'datacenter' which is declared here on type 'RumUserConfiguration'

--- a/packages/logs/README.md
+++ b/packages/logs/README.md
@@ -9,10 +9,11 @@ Datadog browser logs library.
 ### NPM
 
 ```
+import { Datacenter } from '@datadog/browser-core'
 import { datadogLogs } from '@datadog/browser-logs'
 datadogLogs.init({
   clientToken: 'XXX',
-  datacenter: 'us',
+  datacenter: Datacenter.US,
   forwardErrorsToLogs: true,
   sampleRate: 100
 })

--- a/packages/rum/README.md
+++ b/packages/rum/README.md
@@ -7,11 +7,12 @@ Datadog browser rum library.
 ### NPM
 
 ```
+import { Datacenter } from '@datadog/browser-core'
 import { datadogRum } from '@datadog/browser-rum'
 datadogRum.init({
   applicationId: 'XXX',
   clientToken: 'XXX',
-  datacenter: 'us',
+  datacenter: Datacenter.US,
   resourceSampleRate: 100,
   sampleRate: 100
 })

